src/main/java/Duke.java

```diff
@@ -1,10 +1,142 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    private static final int MAX_TASK = 100;
+    private static final Scanner SCANNER = new Scanner(System.in);
+
+    public static void printWelcomeMessage() {
+        System.out.println("-----------------------------------------");
+        System.out.println("!Bot:");
+        System.out.println("Hello! I'm !Bot\nWhat are your orders?");
+        System.out.println("-----------------------------------------");
+    }
+
+    public static void printGoodbyeMessage() {
+        System.out.println("-----------------------------------------");
+        System.out.print("!Bot: ");
+        System.out.println("~~Dead is like a wind, always by my side~~");
+        System.out.println("-----------------------------------------");
+    }
+
+    public static String getInput() {
+        System.out.print("Hung: ");
+        String inputLine = SCANNER.nextLine();
+
+        while (inputLine.trim().isEmpty()) {
+            inputLine = SCANNER.nextLine();
+        }
+
+        return inputLine;
+    }
+
+    public static void printErrorMessage() {
+        System.out.println("-----------------------------------------");
+        System.out.println("!Bot:");
+        System.out.println("Your command is not valid!");
+        System.out.println("Please enter another command!");
+        System.out.println("-----------------------------------------");
+    }
+
+    public static void printTaskList(Task[] tasks) {
+        int taskCount = Task.getNumberOfTask();
+        System.out.println("-----------------------------------------");
+        System.out.println("!Bot:");
+        for(int i = 1; i < taskCount + 1; i = i + 1){
```

Comment:
Maybe could add a space before and after the brackets?
```suggestion
        for (int i = 1; i < taskCount + 1; i = i + 1) {
```